:qa:full-cluster-restart FullClusterRestartIT.testRecovery fails

Likely duplicate of https://github.com/elastic/elasticsearch/issues/31062

The following can be found in the logs:
```
14:53:15   2> REPRODUCE WITH: ./gradlew ':qa:full-cluster-restart:v6.0.0#upgradedClusterTestRunner' -Dtests.seed=A55CB0DC1DA26D51 -Dtests.class=org.elasticsearch.upgrades.FullClusterRestartIT -Dtests.method="testRecovery" -Dtests.security.manager=true -Dtests.locale=de-DE -Dtests.timezone=Europe/Astrakhan -Dcompiler.java=12 -Druntime.java=8
14:53:15 FAILURE 0.06s | FullClusterRestartIT.testRecovery <<< FAILURES!
14:53:15    > Throwable #1: java.lang.AssertionError: mismatch while checking for translog recovery
14:53:15    > testrecovery 0 existing_store done 0
14:53:15    > testrecovery 0 peer           done 0
14:53:15    > testrecovery 1 existing_store done 0
14:53:15    > testrecovery 1 peer           done 0
14:53:15    > testrecovery 2 existing_store done 0
14:53:15    > testrecovery 2 peer           done 0
14:53:15    > testrecovery 3 existing_store done 0
14:53:15    > testrecovery 3 peer           done 0
14:53:15    > testrecovery 4 existing_store done 64
14:53:15    > testrecovery 4 peer           done 0
14:53:15    >  expected:<false> but was:<true>
14:53:15    > 	at __randomizedtesting.SeedInfo.seed([A55CB0DC1DA26D51:64ACC97030F2A7F6]:0)
14:53:15    > 	at org.elasticsearch.upgrades.FullClusterRestartIT.testRecovery(FullClusterRestartIT.java:794)
14:53:15    > 	at java.lang.Thread.run(Thread.java:748)
```

```
./gradlew ':qa:full-cluster-restart:v6.0.0#upgradedClusterTestRunner' \
  -Dtests.seed=A55CB0DC1DA26D51 \
  -Dtests.class=org.elasticsearch.upgrades.FullClusterRestartIT \
  -Dtests.method="testRecovery" \
  -Dtests.security.manager=true
```

Does not reproduce locally.


Build | 20190913124846-031FFA38
-- | --
Log | https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+bwc/BWC_VERSION=6.0.0,nodes=centos-7&&immutable/118/console
Build Scans | https://gradle-enterprise.elastic.co/s/ijnjpreh5erhc

--- a/qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java
+++ b/qa/full-cluster-restart/src/test/java/org/elasticsearch/upgrades/FullClusterRestartIT.java
@@ -674,6 +674,8 @@ public class FullClusterRestartIT extends AbstractFullClusterRestartTestCase {
             ensureGreen(index);
             // Recovering a synced-flush index from 5.x to 6.x might be subtle as a 5.x index commit does not have all 6.x commit tags.
             if (randomBoolean()) {
+                // needs to call a replication action to sync the global checkpoint from primaries to replication.
+                assertOK(client().performRequest(new Request("POST", "/" + index + "/_refresh")));
                 // We have to spin synced-flush requests here because we fire the global checkpoint sync for the last write operation.
                 // A synced-flush request considers the global checkpoint sync as an going operation because it acquires a shard permit.
                 assertBusy(() -> {
@@ -688,7 +690,10 @@ public class FullClusterRestartIT extends AbstractFullClusterRestartTestCase {
                 });
             } else {
                 // Explicitly flush so we're sure to have a bunch of documents in the Lucene index
-                assertOK(client().performRequest(new Request("POST", "/_flush")));
+                Request flushRequest = new Request("POST", "/" + index + "/_flush");
+                flushRequest.addParameter("force", "true");
+                flushRequest.addParameter("wait_if_ongoing", "true");
+                assertOK(client().performRequest(flushRequest));
             }
             if (shouldHaveTranslog) {
                 // Update a few documents so we are sure to have a translog

